e2e-test/log/src/integrationTest/java/com/amazon/dataprepper/integration/log/ApacheLogFaker.java

```diff
@@ -0,0 +1,56 @@
+/*
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  The OpenSearch Contributors require contributions made to
+ *  this file be licensed under the Apache-2.0 license or a
+ *  compatible open source license.
+ *
+ *  Modifications Copyright OpenSearch Contributors. See
+ *  GitHub history for details.
+ */
+
+package com.amazon.dataprepper.integration.log;
+
+import com.github.javafaker.Faker;
+import com.github.javafaker.Internet;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * This class provides the API to generate fake Apache log.
+ */
+public class ApacheLogFaker {
```

Comment:
I like that you made your own ApacheLogFaker. It might be nice to have a single basic unit test for this log creation since changes here can cause the end to end test to fail. A failing unit test could make it easier to pinpoint the cause of failure.